plugins/ReactNativeStyle/ReactNativeStyle.js

```diff
@@ -102,6 +102,10 @@ class NativeStyler extends React.Component {
   _handleStyleRename(oldName: string, newName: string, val: string | number) {
     var style = shallowClone(this.state.style);
     delete style[oldName];
+    if (newName === '') {
+      this.setState({style});
+      return;
```

Comment:
Thanks you so much, @jhen0409! Finally got it working. Can you check my last commit? Am I on the right path?